model/main.py

```diff
@@ -0,0 +1,304 @@
+import argparse
+import numpy as np
+
+from kerastuner.tuners import RandomSearch
+from models.nn import create_model
+from preprocessing import normalize_and_encode
+from tensorflow import keras
+from utils import set_random_state
+from utils import console
+from utils.dataset import (
+    DataProperties,
+    load_data_from_dir,
+    split_features_label,
+)
+from utils.file import save_results_to_csv
+
+# Set python hash and frameworks' random seed to a comon value
+set_random_state(seed=0)
+
+# Don't change these values
+# Facial features stored in the data, combine facial measurement type and
+# base facial feature to get a string index
+facial_measurement_types = ["mean_", "std_", "sum_"]
+
+base_facial_features = [
+    "face_activity_eyebrow",
+    "face_activity_eyebrow_right",
+    "face_activity_eyebrow_left",
+    "face_activity_mouth_corner",
+    "face_activity_mouth_outer",
+    "face_area",
+    "face_blinks_1s",
+    "face_blinks_5s",
+    "face_com_distance",
+    "face_eye_area",
+    "face_motion_instability",
+    "face_mouth_area",
+    "face_mouth_edges_change",
+    "face_mouth_histogram_change",
+]
+
+# Other indices
+HR_ground_header = "mean_HR_ground"
+rppg_header = "HR_poh2011"
+subject_header = "subject"
+game_type_header = "game"
+
+# Define label and feature names to use
+label_header = "emotional_state"
+
+feature_headers = [
+    "mean_HR_ground",
+    "mean_face_activity_mouth_outer",
+    "mean_face_activity_mouth_corner",
+    "mean_face_eye_area",
+    "mean_face_activity_eyebrow",
+    "mean_face_area",
+    "mean_face_motion_instability",
+    "mean_face_com_distance",
+]
+
+# DON'T CHANGE THESE VALUES, IMPORTANT
+validation_subjects = [408, 402, 418, 934, 960, 908]
```

Comment:
In the original implementation, these values are not used. Should we talk with the guys maybe about this question and others similar to it.